Source/externs.m

```diff
@@ -267,22 +267,50 @@
 
 // Pasteboard Type Globals
 NSString *NSStringPboardType = @"NSStringPboardType";
+NSString *NSPasteboardTypeString = @"NSStringPboardType";
+
 NSString *NSColorPboardType = @"NSColorPboardType";
-NSString *NSFileContentsPboardType = @"NSFileContentsPboardType";
-NSString *NSFilenamesPboardType = @"NSFilenamesPboardType";
+NSString *NSPasteboardTypeColor = @"NSColorPboardType";
```

Comment:
I'm not sure this is doable. Unless in C++, we're likely to get `initializer element is not a compile-time constant` if we try to assign one value to another.